app/adapters/alma_adapter/availability_status.rb

```diff
@@ -12,61 +12,71 @@ def initialize(bib:)
 
     # Returns availability information for each of the holdings in the Bib record.
     def bib_availability
+      sequence = 0
       availability = holdings.each_with_object({}) do |holding, acc|
-        status = holding_status(holding)
-        acc[status[:id]] = status
+        sequence += 1
+        status = holding_status(holding: holding, sequence: sequence)
+        acc[status[:id]] = status if status.nil?
       end
       availability
     end
 
-    # TODO: handle these properties
-    #   "copy_number": item["copy_number"],
-    #   "patron_group_charged": nil,    # TODO: nil
-    #   "status": "???",                # TODO: "Not Charged"
-    # TODO: Confirm that we need all of these values with the Alma implementation.
-    # TODO: Find out if we can get copy_number from Solr (i.e. without hitting ExLibris' API)
-    def holding_status(holding)
+    # Returns availability information for each of the holdings in the Bib record.
+    # Notice that although we return the information by holding, we drill into item
+    # information to get details.
+    def bib_availability_from_items
```

Comment:
This method is brand new: `item_data` makes a call to the Alma API to fetch item-level information that we use when the client asks for a "deep_check".